README.md

```diff
@@ -426,6 +426,7 @@ These can be useful for osint and social engineering.
 - [Hybrid Analysis](https://www.hybrid-analysis.com/) - Free malware analysis service for the community that detects and analyzes unknown threats using a unique Hybrid Analysis technology
 - [IBM X-Force Exchange](https://exchange.xforce.ibmcloud.com/) - Threat intelligence sharing platform enabling research on security threats, aggregation of intelligence, and collaboration with peers
 - [The DFIR Report](https://thedfirreport.com/) - Real Intrusions by Real Attackers, The Truth Behind the Intrusion
+- [Detection.FYI](https://detection.fyi/) - Detects when a Microsoft Cloud App Security reported users were active from an IP address identified as risky by Microsoft Threat Intelligence
```

Comment:
This description is on google but it's a description of a specific post on the website. I cannot find a "perfect" description but I would say `Search Sigma rules` is okay

Reply:
Could you please confirm below description is fine or not?
"Detection.FYI is a website that provides a collection of security detections for a variety of security tools and platforms. Detections are categorized by attack technique so that security teams can easily find detections for the specific threats they are concerned about."

Reply:
No, it's too long. `Search Sigma rules` is okay

Reply:
@DicoderRG any update?